examples/tree-view/src/containers/Node.spec.js

```diff
@@ -1,90 +0,0 @@
-import React from 'react'
-import { shallow } from 'enzyme'
-import ConnectedNode, { Node } from './Node'
-
-function setup(id, counter, childIds, parentId) {
```

Comment:
:hand: Can we replace this file with some tests written with React Testing Library?

Reply:
Did my best, tried to mock reducer actions, but couldn't figure it out

Reply:
@markerikson  @aquaductape why we need to test the reducers and actions in the app? I think testing the component behavior correctly will check the reducers functions in the background (and the components tests will executes our reducers and we should get the corresponding response according to that).
You don't think that we will add some implementation details to the tests? (for a simple example it is ok but can confuse readers later and need to write tests for every reducer and actions in a big app and the bundle will be huge).

References: 
https://kentcdodds.com/blog/testing-implementation-details
https://kentcdodds.com/blog/avoid-the-test-user

Reply:
Thanks for sharing @slim-hmidi, and your concerns are valid. Testing isn't my forte and would like to hear what @markerikson has to say.